docs/dev-environment.md

```diff
@@ -126,6 +126,38 @@ Tip: Alternatively, the application supports connecting to the stage environment
 
 Because Image Builder is more complex for installation, we do not recommend installing it on your local machine right now. Configure connection through HTTP proxy to the stage environment in `config/api.env`. See [configuration example](../config/api.env.example) for an example, you will need to ask someone from the company for real URLs for the service and the proxy.
 
+## Containerized environment
+
+a `docker-compose.yml` file help you to roll up the Provisioning application, including frontend and other services such as sources locally for dev or demo with no extra setup.
+
+### Install
+a docker or podman (with [podman-compose](https://github.com/containers/podman-compose)) is needed, the folder structure should be:
+```
+.
+├── provisioning-backend
+├── provisioning-frontend
+├── sources-api-go
+└── image-builder-frontend
+```
+Run 
+```sh
+$ COMPOSE_PROFILES=migrate docker compose up 
+```
+
+### Profiles
+a compose profile allows you to run a subset of containers, when no profile is given, 
+only provisioning backend, frontend, postgres and redis will run, in this case use staged sources.
+
+Currently there are a few profiles:
+- migrate: migrate provisioning backend, terminates after migration
+- kafka: run kafka with zookeeper, register topics
+- sources: run local sources with postgres db, on first use notice that you will need to run `/script/sources.seed.sh` for seeding your local sources data.
```

Comment:
I would appreciate also a profile for frontend and backend. I will use this to run all the backend-backend services except backend

Reply:
frontend profile is added :)